app/pages/lab/workflows.jsx

```diff
@@ -9,10 +9,11 @@ import Paginator from '../../talk/lib/paginator';
 
 const WorkflowsPage = (props) => {
   const renderWorkflow = ((workflow) => {
+    let progressPercentage = workflow.completeness * 100;
     return (
       <li key={workflow.id}>
         <Link key={workflow.id} to={props.labPath(`/workflows/${workflow.id}`)} className="nav-list-item" activeClassName="active">
-          {workflow.display_name}
+          {workflow.display_name}{' -- '}{`${progressPercentage.toFixed(0)} % Complete`}
```

Comment:
I'm not too keen on this. It makes it look like the percentage is a part of the display name.

Reply:
Fair enough. Turning the list into a table should take care of that.